package.json

```diff
@@ -35,15 +36,14 @@
     "karma-chrome-launcher": "2.0.0",
     "karma-sauce-launcher": "^1.1.0",
     "nwb": "^0.15.6",
-    "preact": "^7.2.0",
+    "preact": "*",
     "semantic-release": "^6.3.2",
     "semistandard": "^9.0.0",
-    "skatejs-web-components": "5.2.1",
     "typescript": "^2.1.4",
     "typescript-formatter": "^5.0.0"
   },
   "peerDependencies": {
-    "preact": "^7.2.0"
+    "preact": "*"
```

Comment:
Preact 8 will be released very shortly, which might cause people some pain - the issue is that peer deps don't update when you bump their versions in this package.json.  So the user will update Skate and then get an error saying it's incompatible.